drivers/crypto/crypto_stm32.c

```diff
@@ -48,6 +48,10 @@ LOG_MODULE_REGISTER(crypto_stm32);
 #define STM32_CRYPTO_TYPEDEF            AES_TypeDef
 #endif
 
+#if defined(CONFIG_SOC_SERIES_STM32H5X)
+#define CRYP_DATATYPE_8B CRYP_BYTE_SWAP
+#endif
```

Comment:
there is  already 
`#define CRYP_DATATYPE_8B                CRYP_BYTE_SWAP`
defined  in ../modules/hal/stm32/stm32cube/stm32h5xx/drivers/include/Legacy/**stm32_hal_legacy.h**

Reply:
done with internal ticket #152066